rebalance/portfolio-page.php

```diff
@@ -15,74 +15,93 @@
 	<div id="primary" class="content-area">
 
 		<?php
-			/*
-			 * Include the Featured Project loop.
-			 * - if featured posts exist, collect their IDs and exclude them from the main loop
-			 */
-			if ( rebalance_has_featured_projects( 1 ) ) {
-				$featured_ids = rebalance_get_featured_project_ids();
-				get_template_part( 'template-parts/section', 'featured' );
-			} else {
-				$featured_ids = null;
-			} ?>
-
-		<main id="main" class="site-main" role="main">
-
-			<?php // Set Up New Query
+		if ( post_password_required() ) :
+				the_content();
```

Comment:
Could we add a small explanatory comment for this line? I think this is a fine method for showing the "this is password protected" message and input box, but it's not that intuitive.

Reply:
Thanks for the suggestion, I've added the comment.